src/main/webapp/WEB-INF/tags/instructor/studentsList/filterBox.tag

```diff
@@ -0,0 +1,72 @@
+<%@ tag description="instructorStudentList - Student filter box" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib tagdir="/WEB-INF/tags/instructor/studentsList" prefix="tisl" %>
+<%@ tag import="teammates.ui.template.InstructorStudentsListFilterCourses" %>
+<%@ attribute name="data" type="teammates.ui.template.InstructorStudentsListFilterBox" required="true" %>
```

Comment:
Try not to use data as an attribute name.
It'll deny access to the page data object on a different scope and "data" doesn't really give any hints at what's inside.